compiler/rustc_middle/src/middle/stability.rs

```diff
@@ -292,6 +303,32 @@ fn skip_stability_check_due_to_privacy(tcx: TyCtxt<'_>, def_id: DefId) -> bool {
     }
 }
 
+// See issue #83250.
+fn get_sugg_for_allocator_api(
```

Comment:
```suggestion
fn suggestion_for_allocator_api(
```
We don't need to abbreviate suggestion here :)